### DISCUSSION TITLE
Feature Request - [disable-zoom] but only for mouse scrolls

### DISCUSSION THREAD

**Discussion by sprengerst** [google/model-viewer#2352] [Q&A]
### Description

Hi, is there an option to disable-zoom only for mouse wheel actions, so a pinch zoom on mobile phones is still working?

Best,
Stefan

**Comment by elalish** [comment]
Sure, just add a wheel event listener and put `preventDefault()` in the callback.

**Reply by sprengerst** [reply]
Thanks. You mean to add it directly to the model-viewer element?

**Reply by elalish** [reply]
Yes.

**Reply by designbyadrian** [reply]
In this example, I'm adding a wheel listener to the element, and cannot block the wheel:

https://codepen.io/designbyadrian/pen/vYzgpzq

**Reply by elalish** [reply]
Ah, I misspoke, and additionally you need to `useCapture` in order to intercept our event before we process it. This works in your example: `mv.addEventListener("wheel", (e) => {e.stopPropagation();}, true);`

Copied from example: https://modelviewer.dev/examples/stagingandcameras/#turnSkybox

**Reply by designbyadrian** [reply]
Yes, it works! Thank you for your time.

**Reply by tobiasfabian** [reply]
I use the following to still allow pinch zoom on (MacBook) trackpad and zooming, while the command key (metaKey) is pressed.

```js
mv.addEventListener('wheel', (event) => {
	if (!(event.metaKey || event.ctrlKey)) {
		event.stopPropagation();
	}
}, true);
```
